src/main/drivers/pwm_output.c

```diff
@@ -148,15 +160,14 @@ static void pwmWriteMultiShot(uint8_t index, uint16_t value)
 
 void pwmWriteMotor(uint8_t index, uint16_t value)
 {
-    if (motors[index] && index < MAX_MOTORS && pwmMotorsEnabled)
+    if (motors[index] && index < MAX_MOTORS && pwmMotorsEnabled) {
         motors[index]->pwmWritePtr(index, value);
+    }
 }
 
 void pwmShutdownPulsesForAllMotors(uint8_t motorCount)
 {
-    uint8_t index;
-
-    for(index = 0; index < motorCount; index++){
+    for(int index = 0; index < motorCount; index++){
```

Comment:
I certainly prefer to declare the variable as close as possible to its use, and in this case restrict scope to the for loop block. Prevents accidentally reusing it.